images/ceph/Makefile

```diff
@@ -121,3 +124,15 @@ csv: $(OPERATOR_SDK) $(YQ) ## Generate a CSV file for OLM.
 
 csv-clean: $(OPERATOR_SDK) $(YQ) ## Remove existing OLM files.
 	@rm -fr ../../cluster/olm/ceph/deploy/* ../../cluster/olm/ceph/templates/*
+
+# list-image creates list of images for offline installation
+list-image:
+	@echo "producing list of images for offline installation";\
+	# remove the file if already exists
+	rm -f $(MANIFESTS_DIR)/images.txt;\
+	awk '/image:/ {print $2}' $(MANIFESTS_DIR)/operator.yaml $(MANIFESTS_DIR)/cluster.yaml | \
+	cut -d: -f2- |\
+	tee -a $(MANIFESTS_DIR)/images.txt && \
+	awk '/quay.io/ || /k8s.gcr.io/ {print $3}' $(MANIFESTS_DIR)/operator.yaml | \
+	cut -d: -f2- |\
+	tr -d '"' | tee -a $(MANIFESTS_DIR)/images.txt
```

Comment:
For future reference, this should be replaced with lines that get the CSI-related images from pkg/operator/ceph/csi/spec.go